Changelogs for Digital Twin Manager Microservice 1024.1.1 (#4304)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1024-1-1-added-c-8-y-origin-to-the-definition-of-c-8-y-generic-asset.md

---
date: ""

> title: "Added c8y_Origin to the definition of c8y_GenericAsset."

Reviewer: ```suggestion
title: "Added c8y_Origin to the definition of c8y_GenericAsset"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"

>       label: "Improvement"

Reviewer 2: ```suggestion
      label: "Fix"
```

Reviewer 2: @lara-c8y doesnt see any reason to highlight this as a seperate changelog an improvement.

component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2728"
version: "1024.1.1"
---

> Added the `c8y_Origin` fragment to the `c8y_GenericAsset` Asset

Reviewer: ```suggestion
The`c8y_Origin` fragment has been added to the `c8y_GenericAsset` asset
```

> Definition to indicate that it is part of the library provided by DTM.

Reviewer: ```suggestion
definition to indicate that it is part of the library provided by Digital Twin Manager.
```

> definition to indicate that it is part of the library provided by Digital Twin Manager.

Reviewer: ```suggestion
definition to indicate that it is part of the library provided by the Digital Twin Manager.
```
